data/catalogs/deltares_data.yml

```
@@ -1291,7 +1352,12 @@ merit_hydro_v1.0:
     paper_ref: Yamazaki et al. (2019)
     source_license: CC-BY-NC 4.0 or ODbL 1.0
     source_url: http://hydro.iis.u-tokyo.ac.jp/~yamadai/MERIT_Hydro
-    source_version: 1.0
+    source_spatial_extent:
```

```suggestion
    processing_notes: create vrt file with gdalbuildvrt
    source_spatial_extent:
```
@DirkEilander you also created some layers yourself right? Any script still or at least mention here the list of layers created by you?